annotator/src/test/java/org/cbioportal/annotator/internal/GenomeNexusImplTest.java

```diff
@@ -186,6 +186,33 @@ public void testPostVsGetAnnotationResults() throws Exception {
         }
     }
 
+    @Test
+    public void testGnomadAnnotationResultsExcluded() throws Exception {
+        // test records that should not have the gnomad records
+        for (AnnotatedRecord record : mockAnnotatedRecordsWithPost) {
+            if (record.getGNOMAD_AF() != null) {
+                Assert.fail("testGnomadAnnotationResultsExcluded(), annotated record includes GNOMAD_AF when it should not, value is: '" + record.getGNOMAD_AF() + "'");
+            }
+        }
+    }
+
+    @Test
+    public void testGnomadAnnotationResultsIncluded() throws Exception {
+        // annotate now with my_variant_info included in enrichmentFields
+        ReflectionTestUtils.setField(annotator, "enrichmentFields", GenomeNexusTestConfiguration.MY_VARIANT_INFO_ENRICHMENT_FIELDS);
+
+        AnnotatedRecord record = annotator.makeMockMyVariantInfoAnnotatedRecord(mockAnnotatedRecordsWithPost.get(0));
+        if (record.getGNOMAD_AF() == null) {
+            Assert.fail("testGnomadAnnotationResultsIncluded(), annotated record does not have GNOMAD_AF value when my_variant_info was included, expected value is: '0.001', value is null");
+        }
+        if (!record.getGNOMAD_AF().equals("4.49569E-4")) {
+            Assert.fail("testGnomadAnnotationResultsIncluded(), expected annotated record to have GNOMAD_AF = '0.001', instead it has: '" + record.getGNOMAD_AF() + "'");
```

Comment:
this testing log message does not reflect the actual tested condition (4.49569E-4 vs. 0.001)

Reply:
Thanks!  It was originally 0.001... obviously this should have been a variable.